### DISCUSSION TITLE
PnP search webpart work for some users, but showing zero results for other. Even they have access to sites.

### DISCUSSION THREAD

**Discussion by AshutoshGDS** [microsoft-search/pnp-modern-search#2705] [Q&A]
I am adding Query Template in PnP - search web part.
Earlier it works fine for everyone, but from Friday it seems like working with some users but for some users its not showing any result.

I check with the search query api, users are able to get the result in API but in this webpart its not available.

**Comment by kasperbolarsen** [comment]
Please provide some context, version, browser, query , data source etc.
Please test the same query + data source using the classic search web parts first in order to ensure if this is a PnP Modern Search issue or a SP/MS search issue.
Thanks

**Comment by AshutoshGDS** [comment]
We are fetching Site Titles based on Site Properties.

Query Template:
**({searchTerms} contentclass:sts_site WebTemplate:GROUP XXXXXXX EnvironmentName:XX) OR ({searchTerms} contentclass:sts_site WebTemplate:GROUP TestProperty:XXX EnvironmentName:XX)**

Browser : Edge & Chrome
Version: V4
Data Source: SharePoint

**Comment by kasperbolarsen** [comment]
Please set up the same in a classic search web part and let us know if the same problem is present there as well

**Comment by AshutoshGDS** [comment]
Yes I tried to do do same query setup in classic search webpart & its working fine & give respected site result with title. So it seems like some issue with Modern PNP search webpart only.

**Reply by AshutoshGDS** [reply]
As per initial debugging, what I found is that user ID with specific domain faces the issue. This might help but not sure what exactly stop working suddenly.
